serverlessworkflow/modules/ROOT/pages/cloud/common/_proc_deploy_sw_kubectl.adoc

```diff
@@ -0,0 +1,174 @@
+[[proc-deploy-sw-application-kubectl]]
+=== Deploying your workflow application using `kubectl`
+
+You can also use `kubectl` command-line interface and plain Kubernetes objects to deploy your workflow application. +
+And, instead of creating `knative` `yaml|json` descriptors, you can leverage the Quarkus Kubernetes extension and
+Kogito Knative add-on to generate the descriptors.
+
+.Prerequisites
+* Kogito Workflow application ready to use.
+* `kubectl` command-line tool is installed.
+* (Optional) Quarkus CLI is installed +
```

Comment:
why do you need the Quarkus CLI in that section ?

Reply:
it can be used in during this section to add the extension and build the app, it is not described the exact steps.

Reply:
If you are not mentioning it, then remove it from prerequisites. It could confuse the user ...